### PR TITLE
String tweaks in response to feedback; add fingerprint to template

### PR DESCRIPTION
**Pull request by eloquence**
## Status

Ready for review. NB: This change is targeting SecureDrop 1.6.0, and does not need to be backported to 1.5.0.

## Description
Misc. changes based on translator feedback, plus a change to the way that the submission key fingerprint is presented in the Source Interface:

- Add '.' to some messages [1] [2] [3]
- Simplify and shorten validation warnings [4]
- Tweak error message for bad form data case [5]
- Avoid idiosyncratic ampersand usage per [6]
- Pull closing tag into source string [7]
- Change "Security Setting" to "Security Level", which is what Tor Browser now uses [8]
- Desktop versions->desktop version [9]
- Add submission key fingerprint to the "Why Journalist Key" template, to avoid convoluted instructions there; remove reference to UFO incident

(This change was done in collaboration with @rmol and only possible thanks to the feedback provided by many translators.)

[1] https://weblate.securedrop.org/translate/securedrop/securedrop/en/?checksum=8e012ca12dd735dc#comments
[2] https://weblate.securedrop.org/translate/securedrop/securedrop/en/?checksum=e2291f9285fe59af#comments
[3] https://weblate.securedrop.org/translate/securedrop/securedrop/en/?checksum=782cc89459efa038#comments
[4] https://weblate.securedrop.org/translate/securedrop/securedrop/en/?checksum=4422f6d628edc7de#comments
[5] https://weblate.securedrop.org/translate/securedrop/securedrop/en/?checksum=511d2e2028f7fe91#comments
[6] https://weblate.securedrop.org/translate/securedrop/securedrop/en/?checksum=aa334ee40ac41bf#comments
[7] https://weblate.securedrop.org/translate/securedrop/securedrop/en/?checksum=486d7cd4ec4458de#comments
[8] https://weblate.securedrop.org/translate/securedrop/securedrop/en/?checksum=8a1dbec76ada33ea#comments
[9] https://weblate.securedrop.org/translate/securedrop/securedrop/en/?checksum=2f7252b4a244f071#comments

## Test plan

For the fingerprint changes:
- Spin up this branch in the development environment
- [ ] Observe that the fingerprint is rendered correctly on `http://localhost:8080/why-journalist-key` and that the instructions make sense.

## Checklist

- [x] `docs-lint` passes
- [ ] `make test` passes (waiting for CI)

### CODE REVIEW

**Comment by eloquence** [comment]
(@rmol reviewed a prior version of this PR, and contributed to it, see here for history: https://github.com/freedomofpress/securedrop/pull/5391#pullrequestreview-452866766)

**Review by gonzalo-bulnes — COMMENTED** [review]
For what is worth, the changes match the comments I've seen in Weblate and I see no typos. :+1:

File: docs/source.rst [hunk 1]
```diff
@@ -164,7 +164,7 @@ similar to this screenshot:
    The "Safest" setting disables the use of JavaScript on every page you visit
    using Tor Browser, even after a browser restart. This may cause other
    websites you visit using Tor Browser to no longer work correctly, until
-   you adjust the Security Setting again. We recommend keeping the setting at
+   you adjust the Security Level again. We recommend keeping the setting at
```
Comment (zenmonkeykstop) [inline docs/source.rst:167]: Capitalization is inconsistent with the first reference at line 145.

**Review by zenmonkeykstop — APPROVED** [review]
One minor nit, but looking good otherwise. Approving and thanks to all involved in these changes!

**Pull request merged**